app/js/services/resultatService.js

```diff
@@ -2,20 +2,25 @@
 
 angular.module('ddsApp').service('ResultatService', function($http, $modal, droitsDescription) {
 
+    var DROITS_POTENTIELS = droitsDescription.prestationsNationales.concat(
+        _.flatten(_.pluck(droitsDescription.partenairesLocaux, 'prestations'))
```

Comment:
Pure JS equivalent:
```js
Array.prototype.concat.call(
    droitsDescription.prestationsNationales,
    droitsDescription.partenairesLocaux.map(function(partenaire) { return partenaire.prestations })
);
```

(no strong opinion here, I'm less familiar with `_` and `pluck` was not very explicit, but it is terser)